internal/graph/check.go

```diff
@@ -344,6 +344,8 @@ func (c *LocalChecker) ResolveCheck(
 		return nil, fmt.Errorf("relation '%s' undefined for object type '%s'", relation, objectType)
 	}
 
+	span.SetAttributes(attribute.String("rewrite", tuple.ToObjectRelationString(object, relation)))
```

Comment:
This attribute is misleading to me, because it's not a rewritten value. The only scenario where we evaluate a rewritten value is with the `computed_userset`. Maybe `relationship` or `object_relation` is a more aptly named attribute here.

Reply:
Okay then `checkRewrite` should be called `checkRelationship`?